app/tydom/TydomClient.py

```diff
@@ -52,6 +54,56 @@ def __init__(
             self.cmd_prefix = ""
             self.ping_timeout = None
 
+    @staticmethod
+    def getTydomCredentials(login: str, password: str, macaddress: str):
+        """get tydom credentials from Delta Dore"""
+        try:
+            response = requests.get("https://deltadoreadb2ciot.b2clogin.com/deltadoreadb2ciot.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_AccountProviderROPC_SignIn")
+        
+            json_response = response.json()
+            response.close()
+            signin_url = json_response["token_endpoint"]
+            logger.debug("signin_url : %s", signin_url)
```

Comment:
Unfortunately log debugs are never displayed despite setting an env var LOG_LEVEL=debug.

It's because the logger configuration occurs **after**: [See here](https://github.com/fmartinou/tydom2mqtt/blob/master/app/main.py#L28-L37)

So maybe this code should be moved sooner (right after the log configuration has been set?)

Between L61 and L62 of the Configuration.py script?